src/main/java/org/wildfly/security/keystore/LdapKeyStore.java

```diff
@@ -102,7 +103,7 @@ public LdapKeyStore build() {
                     filterAlias, filterCertificate, filterIterate, createPath, createRdn, createAttributes, aliasAttribute,
                     certificateAttribute, certificateType, certificateChainAttribute, certificateChainEncoding,
                     keyAttribute, keyType);
-            return new LdapKeyStore(spi, null, null);
+            return new LdapKeyStore(spi, new WildFlyElytronProvider(), "LdapRealm");
```

Comment:
I know that instantiating WildFlyElytronProvider is a bit expensive and I wanted to create new dummy Provider, but the constructor is protected.